CHANGELOG.md

```diff
@@ -1,4 +1,52 @@
-# Changelog
+# CHANGELOG
+
+## 1.25.0 (2022-04-08)
+
+MLflow 1.25.0 includes several major features and improvements:
+
+Features:
+
+- pyfunc.spark_udf: conda env dir / cache dir isolation and NFS optimization  (#5561, @WeichenXu123)
+- Introduce an mlflow.artifacts.download_artifacts() API, equivalent to the preexisting CLI (#5585, @dbczumar)
+- Update UI (#5577, @harupy)
```

Comment:
@harupy @BenWilson2 Can we summarize the key UI updates from the copybara import?

Reply:
I just checked https://github.com/mlflow/mlflow/pull/5577 and it doesn't contain any user-facing changes.

Reply:
It contains the changes for cross-experiment run comparison, but I don't think we need to mention it.

Reply:
Maybe jquery removal should be highlighted?

Reply:
I think we can remove this entry and put it in the "small updates" section.

Reply:
No, it's not. Putting in the small updates section sounds good

Reply:
k will do!